assets/scss/components/_togglePill.scss

```diff
@@ -116,6 +116,10 @@ Checkbox form elements styled as toggle buttons.
 	-webkit-tap-highlight-color: rgba(0,0,0,0);
 }
 
+.toggleButton-label--radio {
+	padding: $space $space*1.5;
+}
+
```

Comment:
I just increased the `padding`. Should I change the color to the blue in the mocks?

Reply:
Change the color in `mup-web`. The blue is a non-standard style I'm going to get the design team to make a decision on after this goes out.